src/main/resources/assets/securitycraft/lang/ko_kr.json

```diff
@@ -0,0 +1,809 @@
+	"help.securitycraft.camera_monitor.info": "카메라 모니터는 최대 30대의 보안 카메라의 경계를 위해 실시간 피드를 볼 때 사용하는 휴대용 아이템입니다. 카메라 모니터가 있는 보안 카메라를 마우스 오른쪽으로 클릭하여 바인딩한 다음, 허공를 마우스 오른쪽을 클릭하고 카메라 청크가 로드되어 있는 동안 GUI에서 카메라를 선택하여 피드를 확인합니다. 카메라 모니터로 다시 마우스 오른쪽을 클릭하거나 카메라 모니터의 GUI를 열고 해당 \"X\" 버튼을 클릭하여 보안 카메라 바인딩을 해제합니다.",
+	"help.securitycraft.sc_manual.info": "세큐리티크래프트 설명서는 세큐리티크래프트가 추가하는 모든 블록 및 항목에 대한 기본 정보를 보여줍니다. 자세한 설명은 에서 FTB Wiki에서 https://ftb.gamepedia.com/SecurityCraft를 참조하시기 바랍니다.",
+	"help.securitycraft.universal_owner_changer.info": "만능 주인 변경기 는 블록의 주인을 변경할 수 있습니다. 이렇게 하려면 모루의 아이템 이름을 블록의 새 주인의 이름으로 변경합니다. 그러면 주인과 함께 블록 중 하나를 마우스 오른쪽을 클릭하면 변경됩니다. 활성화된 경우 이름 없는 변경기로 마우스 오른쪽을 클릭하여 주인이 없는 블록을 클레임할 수 있습니다.",
+	"help.securitycraft.universal_block_reinforcer_lvl1.info": "만능 블록 강화기는 수많은 바닐라 블록을 강화하여 세큐리티크래프트 변형으로 만듭니다. 이 작업을 수행하려면 만능 블록 강화기를 마우스 오른쪽으로 클릭하고 바닐라 블록을 상단 슬롯에 넣은 다음 GUI를 닫습니다. 강화 블록을 다시 바닐라 블록으로 변환하려면 하단 슬롯을 사용합니다. 1단계는 300개를 사용합니다. 크리에이티브 모드를 벗어나는 블록을 모두 파괴하여 보강합니다.",
+	"help.securitycraft.universal_block_reinforcer_lvl2.info": "만능 블록 강화기는 수많은 바닐라 블록을 강화하여 세큐리티크래프트 변형으로 만듭니다. 이 작업을 수행하려면 만능 블록 강화기를 마우스 오른쪽으로 클릭하고 바닐라 블록을 상단 슬롯에 넣은 다음 GUI를 닫습니다. 강화 블록을 다시 바닐라 블록으로 변환하려면 하단 슬롯을 사용합니다. 2단계는 2700개를 사용합니다. 크리에이티브 모드를 벗어나는 블록을 모두 파괴하여 보강합니다.",
+	"help.securitycraft.universal_block_reinforcer_lvl3.info": "만능 블록 강화기는 수많은 바닐라 블록을 강화하여 세큐리티크래프트 변형으로 만듭니다. 이 작업을 수행하려면 만능 블록 강화기를 마우스 오른쪽으로 클릭하고 바닐라 블록을 상단 슬롯에 넣은 다음 GUI를 닫습니다. 강화 블록을 다시 바닐라 블록으로 변환하려면 하단 슬롯을 사용합니다. 3단계는 무제한 사용합니다. 크리에이티브 모드를 벗어나는 블록을 모두 파괴하여 보강합니다.",
+	"help.securitycraft.briefcase.info": "서류가방은 12개의 슬롯이 들어 있는 키패드 잠금식 휴대용 저장장치입니다. 조작한 후 마우스 오른쪽을 클릭하여 그때부터 사용할 4자리 암호가 설정됩니다. 다시 마우스 오른쪽을 누르고 항목과 상호 작용할 코드를 삽입합니다. 서류 가방의 코드를 제거하려면 만는 열쇠 변경기로 함께 만듭니다.",
+	"help.securitycraft.universal_key_changer.info": "만능 열쇠 변경기는 당신이 가지고 있는 암호로 보호된 블록의 암호를 변경할 수 있게 해줍니다. 암호로 보호된 블록을 마우스 오른쪽으로 눌러 GUI를 연 다음 원하는 새 암호를 입력합니다.",
+	"help.securitycraft.secret_sign_item.info": "비밀 표지판은 바닐라 표지판과 같이 작동하는데, 표지판 주인만이 그 위에 있는 텍스트를 볼 수 있다는 약간의 차이가 있습니다. 아무도 그 블록을 어길 수 없도록 합니다!",
+	"help.securitycraft.sentry.info": "보초는 오른쪽 클릭으로 전환할 수 있는 3가지 모드를 가지고 있습니다. 유휴(그것은 결코 공격하지 않을 것입니다.), 위장(그 범위에서 적의 플레이어만을 공격할 것입니다.), 공격(적대적인 몹과 적들을 공격할 것입니다.) 완전하고 불투명한 블록(예: 돌이나 유리나 계단이 아님)이 들어 있는 변장 모듈로 마우스 오른쪽 버튼을 클릭하면 센트리가 변장하게 됩니다. 모듈을 직접 전환하거나 만능 블록 수정기를 사용하여 마우스 오른쪽 버튼을 클릭하여 추출할 수 있습니다. 보초를 제거하려면 보초를 마우스 오른쪽을 클릭합니다.",
+	"help.securitycraft.crystal_quartz_item.info": "수정 석영은 수정 석영 블록을 만드는 데 사용됩니다.",
+
+	"gui.securitycraft:scManual.disabled": "사용안함",
+	"gui.securitycraft:scManual.recipe.keycard_lv1": "사용안함",
+	"gui.securitycraft:scManual.recipe.keycard_lv2": "사용안함",
+	"gui.securitycraft:scManual.recipe.keycard_lv3": "사용안함",
+	"gui.securitycraft:scManual.recipe.keycard_lv4": "사용안함",
+	"gui.securitycraft:scManual.recipe.keycard_lv5": "사용안함",
+	"gui.securitycraft:scManual.recipe.limited_use_keycard": "사용안함",
+	"gui.securitycraft:scManual.intro.1": "세큐리티크래프트 시작하기",
+	"gui.securitycraft:scManual.intro.2": "Geforce, bl4ckscor3",
```

Comment:
Translate and add
`    "gui.securitycraft:scManual.author": "Translated by mindy15963",`
after this line